modules/client-common/CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ TEMPLATE:
 
 ## [UPCOMING]
 ### Added
+- Support for baseMainnet network
+## 1.3.0-rc0
+### Added
 - Support for baseGoerli network
```

Comment:
As mentioned in https://github.com/aragon/sdk/pull/265/files#r1282826187, this might not be correct.

Reply:
nope, this one is correct, I added the previous version to the changelog